raisinbread/instruments/eeg_acquisition_form.linst

```diff
@@ -0,0 +1,88 @@
+title{@}EEG Acquisition Form
+date{@}Date_taken{@}Date of Administration{@}{@}
+static{@}Candidate_Age{@}Candidate Age (Months)
+static{@}Window_Difference{@}Window Difference (+/- Days)
+select{@}Examiner{@}Examiner{@}
+textarea{@}general_comments{@}General session notes, any important details (e.g. technical error, issue with net, etc.)
+header{@}{@}<b><h3>General information</h3></b>
+text{@}research_staff_one{@}Research staff 1 initials
+text{@}research_staff_two{@}Research staff 2 initials
+text{@}research_staff_three{@}Research staff 3 initials
+text{@}session_testing_room{@}Testing room information
+select{@}behavioural_assistant_present{@}Is there a behavioural assistant?{@}'yes'=>'Yes'{-}'no'=>'No'
+text{@}behavioural_assistant{@}Behavioural assistant initials
+select{@}subject_medication{@}Is subject under medication?{@}'yes'=>'Yes'{-}'no'=>'No'
+textarea{@}subject_medication_dosage{@}Describe the medication and dosage
+select{@}head_measurement_unit{@}What is the unit used for head measurement?{@}'cm'=>'Centimeters (cm)'{-}'in'=>'Inches (in)'
```

Comment:
Please add an option : "No measurement taken" -- it's really only for pediatric studies that they measure.
please add a rule:  if "no measurement taken" is selected, the following field (for the number) is optional.